src/components/step/StepMarkdown/StepMarkdown.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {useEffect} from 'react'
+import MarkdownIt from 'markdown-it';
+import parse from 'html-react-parser'
+
+export interface StepMarkdownProps {
+  text: string;
+  inline?: boolean;
+}
+
+export default function (props: StepMarkdownProps) {
```

Comment:
This is definitely fine for now, but as we move toward possibly allowing editing of custom views dynamically (like steps), I can imagine making this more generic (e.g. calling it Markdown) and moving it up to the presentational components